pkg/controllers/management/clusterdeploy/clusterdeploy.go

```diff
@@ -92,6 +93,10 @@ func (cd *clusterDeploy) sync(key string, cluster *v3.Cluster) (runtime.Object,
 		return nil, nil
 	}
 
+	if cluster.Status.Driver == v32.ClusterDriverImported && clusterconnected.Connected.IsFalse(cluster) {
```

Comment:
Can a log be added here like "cluster [clusterName] deploy skipped: cluster is disconnected."

Reply:
It will log a lot then.